docs/contributor/validator.md

```
@@ -1,534 +1,541 @@
-- `fmt.Printf` → stdout → captured as CTRF evidence
-- `return nil` → exit 0 → passed
-- `return errors.*` → exit 1 → failed (message written to termination log)
-- `return validators.Skip(reason)` → exit 2 → skipped
+`slog.*` → stderr → streamed live. `fmt.Printf` → stdout → captured
+as CTRF evidence. `return nil` → 0, `return error` → 1,
+`return validators.Skip(reason)` → 2.
 
-## Directory Layout
+### Directory layout
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)